lib/media/streaming_engine.js

```diff
@@ -251,6 +251,7 @@ shaka.media.StreamingEngine = class {
       this.abortOperations_(state).catch(() => {});
       this.mediaStates_.delete(ContentType.TEXT);
     }
+    this.currentTextStream_ = null;
```

Comment:
Oh, wow, that certainly should have been there.  :-)

This member is new in v3.0 if I recall correctly.